modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -580,44 +581,55 @@ private void _initSystemObjectDefinition(
 		_serviceRegistrationsMap.computeIfAbsent(
 			jaxRsApplicationDescriptor.getRESTContextPath(),
 			key -> _registerExceptionMappers(
-				jaxRsApplicationDescriptor.getApplicationName(),
-				objectDefinition));
+				jaxRsApplicationDescriptor.getApplicationName()));
 	}
 
 	private List<ServiceRegistration<?>> _registerExceptionMappers(
-		String jaxRsApplicationName, ObjectDefinition objectDefinition) {
+		String jaxRsApplicationName) {
 
 		return TransformUtil.transform(
-			Arrays.asList(
-				new ObjectAssetCategoryExceptionMapper(),
-				new ObjectEntryCountExceptionMapper(),
-				new ObjectEntryManagerHttpExceptionMapper(),
-				new ObjectEntryStatusExceptionMapper(_language),
-				new ObjectEntryValuesExceptionMapper(_language),
-				new ObjectRelationshipDeletionTypeExceptionMapper(_language),
-				new ObjectValidationRuleEngineExceptionMapper(
+			Arrays.<Supplier<ExceptionMapper<?>>>asList(
```

Comment:
in a spare time I'd like to ask you for the details of the approach, as this function is little hard to understand for me. But all the tests of API Builder are passing now, so I'll forward ✅

Reply:
hey @magjed4289 this is a change to a private method... I think that semantic change is not related.